### PR TITLE
docs for additional-node-infos

### PR DESCRIPTION
**Pull request by al-r3**
Add documentation about the additional-node-infos directory

### CODE REVIEW

File: docs/source/network-map.rst [hunk 1]
```diff
@@ -35,4 +43,15 @@ Network Map service REST API:
...
+
+The ``additional-node-infos`` directory
+---------------------------------------
+Each Corda node reads, and continuously polls, the files contained in a directory named ``additional-node-infos`` inside the node base directory.
+
+Nodes expect to find a serialized ``SignedData<NodeInfo>`` object, the same object which is sent to network map server.
+
+Whenever a node starts it writes on disk a file containing its own ``NodeInfo``, this file is called ``nodeInfo-XXX`` where ``XXX`` is a long string.
+
+Hence if an operators wants node A to see node B they can pick A's ``NodeInfo`` file from A base directory and drop it into B's ``additional-node-infos`` directory.
```
Comment (gendal) [inline docs/source/network-map.rst]: surely the other way round?

Comment (al-r3) [reply docs/source/network-map.rst]: yup, nice catch.
It is somewhat likely they'll have to do that both ways.

Comment (gendal) [reply docs/source/network-map.rst]: oh - also just spotted: operator, not operators.

File: docs/source/network-map.rst [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Network Map
 ===========
 
+The network map stores a collection of ``NodeInfo`` objects, each representing another node with with one can interact.
```
Comment (shamsasari) [inline docs/source/network-map.rst]: typo: double with

Comment (shamsasari) [inline docs/source/network-map.rst]: "the node" instead of "one"

File: docs/source/network-map.rst [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+There 2 sources from which a Corda node can retrieve ``NodeInfo`` objects:
```
Comment (shamsasari) [inline docs/source/network-map.rst]: "two"

**Pull request merged**